test/support/page_objects/common.js

```diff
@@ -68,6 +68,13 @@ export default class Common {
     }
   }
 
+  waitUntilUrlFound(path) {
```

Comment:
How about `waitUntilUrlIncludes` to help indicate that this isn't looking for an exact match?